.github/workflows/deploy.yml

```diff
@@ -0,0 +1,98 @@
+name: github pages
+
+on: [push, pull_request]
+
+jobs:
+  deploy-landing-page:
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
```

Comment:
Question: Is this `uses` supposed to be part of the same block as `- name: Set output`?